notebooks/test_2.ipynb

```diff
@@ -0,0 +1,59 @@
+{
```

Comment:
<p>(reviewer): I see the capitalisation change.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/moaatt2/reviewnb-test/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>